specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-08-01/workbooks_API.json

```diff
@@ -680,7 +683,7 @@
       "in": "path",
       "required": true,
       "type": "string",
-      "description": "The name of the Application Insights component resource.",
+      "description": "The name of the workbook resource.",
```

Comment:
why change the description, it conflicts with another file which contains the same parameter, you need to change both if them , you might need to check all the input files under package-2022-02-01

Reply:
workbooks is related to application insights but isn't tied to application insights for a long time now